pkg/api/testapi/testapi.go

```diff
@@ -71,9 +74,16 @@ func init() {
 			internalGroupVersion: extensions.SchemeGroupVersion,
 		}
 	}
+	if _, ok := Groups["testgroup"]; !ok {
+		Groups[testgroup.SchemeGroupVersion.Group] = TestGroup{
+			externalGroupVersion: unversioned.GroupVersion{Group: testgroup.SchemeGroupVersion.Group, Version: latest.GroupOrDie(testgroup.SchemeGroupVersion.Group).GroupVersion.Version},
+			internalGroupVersion: testgroup.SchemeGroupVersion,
+		}
+	}
 
 	Default = Groups[api.SchemeGroupVersion.Group]
 	Extensions = Groups[extensions.SchemeGroupVersion.Group]
+	Testgroup = Groups[testgroup.SchemeGroupVersion.Group]
```

Comment:
close. You just need a local package variable, right? No need to modify that map.

```go
var testGroup = testapi.TestGroup{
...
}
```